src/main/java/oop/House.java

```diff
@@ -0,0 +1,73 @@
+package main.java.oop;
+
+
+public class House {
+    int roomCount;
+    String address;
+    String color;
+    String roofColor;
+    boolean serenaOff;
+
+    public void serena() {
```

Comment:
Method should start from a verb. 'turnOfSerena()' will be ok. but not just serena. have you read java convention? please read it ;)